cpp/mrc/src/internal/data_plane/data_plane_resources.cpp

```diff
@@ -363,6 +378,46 @@ std::shared_ptr<ucxx::Request> DataPlaneResources2::am_recv_async(std::shared_pt
     return request;
 }
 
+uint64_t DataPlaneResources2::get_next_object_id()
+{
+    // TODO(Peter): Handle overflow?
```

Comment:
We can ignore overflow. The max is 18,446,744,073,709,551,615 messages. At our current throughput of 11k/sec, that would take decades to hit that max value.

Reply:
Yeah, I wasn't sure how fast they could go. Removed comment via https://github.com/nv-morpheus/MRC/pull/437/commits/3e0cf32dfded9f96b9d539430f751d0bac47a1db .